classifier_and_summarizer_interface.py

```diff
@@ -40,20 +40,23 @@ def preprocess_contents(self):
     def get_stampified_content(self):
         ''' returns the list of stamp pages'''
         # pre-process the contents first
-        self.preprocess_contents()
+        self._preprocess_contents()
```

Comment:
ExtractorOutputPreprocessor should be in classification/ and not summarization/

Reply:
I was conflicted on where to place this since extractor output pre-processor's output was used for both classifier and summarizer . So do you think it'd make sense to place it in `stampify/` itself rather than either `classification/` or `summarization/` ?

Reply:
Yeah, thats also fine. Under a new module?

Reply:
I was thinking of leaving it as a script under `stampify/` itself. WDYT?

Reply:
I guess i'll do this in a separate PR. #57 is dependent on this script. Will do that once #57 is merged